ndvi_forecast/1_download_data.ipynb

```diff
@@ -250,7 +252,7 @@
     "    \n",
     "    # Submit Satellite Job\n",
     "    try:\n",
-    "        print(\"Queuing satellite job... \", end=\"\", flush=True)\n",
+    "        print(\"Queuing satellite job for boudary '{}'. \".format(boundary_obj.id), end=\"\", flush=True)\n",
```

Comment:
Let's convert all of the `.format`s into f strings.

Reply:
Done